src/main/java/io/pinecone/clients/Pinecone.java

```diff
@@ -22,48 +22,6 @@ private Pinecone(PineconeConfig config, ManageIndexesApi manageIndexesApi) {
         this.manageIndexesApi = manageIndexesApi;
     }
 
-    public static class Builder {
```

Comment:
Uh... I don't know why these removals of `Builder` are in here...I'll look into it